javascript/src/test/components/Nav/AppNavbar.test.js

```diff
@@ -31,6 +31,18 @@ describe("AppNavbar tests", () => {
     const brandElement = getByText(/Mapache Search/);
     expect(brandElement).toBeInTheDocument();
   });
+  test("should have the correct links in the navbar", () => {
+    const history = createMemoryHistory();
+    const { getByText } = render(
+      <Router history={history}>
+        <AppNavbar />
+      </Router>
+    );
+
+    const userInfoLink = getByText(/Profile/);
+    expect(userInfoLink.href).toMatch("/profile");
+  });
+  
```

Comment:
I wonder if this test is failing because another team's change to remove the `Profile` link and replace it with linking the user's name to the profile (i.e. making `Welcome, Chris Gaucho` the link to the Profile) has already been merged?

Reply:
Fiexed